test/gtest/uct/ib/test_ib_md.cc

```diff
@@ -139,12 +135,6 @@ bool test_ib_md::has_ksm() const {
 bool test_ib_md::check_umr() const {
 #if HAVE_DEVX
     return has_ksm();
-#elif HAVE_EXP_UMR
```

Comment:
Is the above `#if HAVE_DEVX` really needed, considering it's already in `has_ksm()`?

Reply:
I'll remove ```#if HAVE_DEVX``` here and remove ```check_umr``` function.

Reply:
They are `protected`. Overridden in sub-classes?

Reply:
there's no sub-classes inherited from it.